facade/registry.go

```diff
@@ -35,9 +37,37 @@ func (f *Facade) SetRegistryImage(ctx datastore.Context, rImage *registry.Image)
 	if err := f.registryStore.Put(ctx, rImage); err != nil {
 		return err
 	}
-	if err := f.zzk.SetRegistryImage(rImage); err != nil {
+	uuid, err := f.zzk.SetRegistryImage(rImage)
+	if err != nil {
 		return err
 	}
+	svcs, err := f.GetAllServices(ctx)
```

Comment:
I can switch over to that.  I originally had it in there, but switched to `GetAllServices` because I thought I had to cover the swapping repos case in here, but I ended up covering that in the service facade.